src/components/molecules/DonatePopUp/DonatePopUp.tsx

```diff
@@ -1,26 +1,32 @@
-import React, { useState } from "react";
-import InformationLeftColumn from "components/organisms/InformationLeftColumn";
-import InformationCard from "../InformationCard";
-import "./DonatePopUp.scss";
+import React, { useRef } from "react";
+
 import {
   BURNING_MAN_DONATION_SITE,
   BURNING_MAN_DONATION_TEXT,
   BURNING_MAN_DONATION_TITLE,
-} from "../../../../src/settings";
+} from "settings";
+
+import {
+  InformationLeftColumn,
+  InformationLeftColumnControls,
+} from "components/organisms/InformationLeftColumn";
+
+import InformationCard from "components/molecules/InformationCard";
+
+import "./DonatePopUp.scss";
 
 export const DonatePopUp = () => {
-  const [isLeftColumnExpanded, setIsLeftColumnExpanded] = useState(false);
+  const controlsRef = useRef<InformationLeftColumnControls>(null);
+
+  const { toggleExpanded } = controlsRef.current ?? {};
 
   return (
-    <InformationLeftColumn
-      venueLogoPath={"heart"}
-      isLeftColumnExpanded={isLeftColumnExpanded}
-      setIsLeftColumnExpanded={setIsLeftColumnExpanded}
-    >
+    <InformationLeftColumn ref={controlsRef} iconNameOrPath="heart">
```

Comment:
FYI: This pattern allows a direct parent to gain access to 'internal state/controls' as defined/provided by a component.